config/api.yml

```diff
@@ -621,6 +621,8 @@
         :identifier: embedded_configuration_script_source_delete
       - :name: create
         :identifier: embedded_configuration_script_source_add
+      - :name: refresh
+        :identifier: embedded_configuration_script_source_edit
```

Comment:
@jntullo is this what you're looking for? https://github.com/ManageIQ/manageiq/pull/14664